### PR TITLE
Update Pepipost/Add Netcore Cloud

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Review by ApCoder123 — COMMENTED** [review]
This seems to be a rebrand of `pepipost.com`, which as already listed. However, the doc link for pepipost redirects to the doc link given here for Netcore Cloud (Email services). I think that it is probably better to remove the Pepipost entry/rename it to Netcore Cloud, rather than include both.

**Comment by 40504** [comment]
As is now netcorecloud.com

File: entries/n/netcorecloud.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Netcore Cloud": {
+    "domain": "netcorecloud.com",
```
Comment (ApCoder123) [inline entries/n/netcorecloud.com.json:3]:
```suggestion
    "domain": "netcorecloud.com",
    "additional-domains": [
      "pepipost.com"
    ],
```

The `pepipost.com` domain still seems to be in use despite the renaming.

File: entries/n/netcorecloud.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Netcore Cloud": {
```
Comment (ApCoder123) [inline entries/n/netcorecloud.com.json]:
```suggestion
  "Netcore Cloud Email API": {
```
2FA only applies to this specific product.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: entries/n/netcorecloud.com.json (hunk 2 above)
Comment (40504) [reply entries/n/netcorecloud.com.json]: why "Netcore Cloud Email API": when everywhere is "Netcore Cloud"?

Comment (ApCoder123) [reply entries/n/netcorecloud.com.json]: The doc page provided is for their email API:
image

whereas other netcore cloud products (see https://netcorecloud.com/login) don't appear to support 2FA.

**Review by ApCoder123 — APPROVED** [review]

File: entries/n/netcorecloud.com.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "additional-domains": [
+      "pepipost.com"
+    ],
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "https://emaildocs.netcorecloud.com/docs/two-factor-authentication-2fa",
+    "keywords": [
+      "communication"
```
Comment (Carlgo11) [inline entries/n/netcorecloud.com.json:13]: Just like in #6017, marketing would be the proper category.
```suggestion
      "marketing"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/n/netcorecloud.com.json (hunk 3 above)
Comment (Carlgo11) [reply entries/n/netcorecloud.com.json:13]: I'll change it in that PR instead.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**